src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -400,7 +400,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 });
             }
 
-            if(env.IsDevelopment() || Configuration.GetValue<bool>("enableSwagger"))
```

Comment:
I'm not 100%, but I think `env.IsDevelopment()` allows Swagger to work when you're running Admin locally.

Reply:
Yup - but it's also set to true in appsettings.Development.json, so I decided this was a redundant check

Reply:
I was sure Swagger wouldn't work locally without this, but tested it and it works fine.

This `env.IsDevelopment()` could also be removed from the same line for Content and Data too.